### PR TITLE
bugfix/ATC-191

### PR DESCRIPTION
**Pull request by Dabea**
This will fixes the bug where parseElevation() returns Nan

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -73,3 +74,4 @@
 - Aircraft strips show arrival airport in uppercase [#108](https://github.com/n8rzz/atc/issues/108)
 - Updates `FixCollection.findFixByName()` to accept upper, mixed, or lower case fix name [#109](https://github.com/n8rzz/atc/issues/109)
 - Switching to a previously loaded airport does not clear previous airport fixes [#115](https://github.com/n8rzz/atc/issues/115)
+- Fixes `parseElevation()` so that it dose not return NaN when it is given the string `'Infinity'` [#191] (https://github.com/n8rzz/atc/issues/191)
```
Comment (n8rzz) [inline CHANGELOG.md]: I think you meant to use `does` here and not `dose`

Comment (n8rzz) [reply CHANGELOG.md]: could you also add a link to the origin zlsa issue here too?  see this PR for an example of how that should be formatted: https://github.com/n8rzz/atc/pull/198/files

File: src/assets/scripts/utilities/unitConverters.js [hunk 2]
```diff
@@ -270,6 +270,10 @@ export const parseElevation = (elevation) => {
         return parseFloat(elevation);
     }
 
+    if (elevation === 'Infinity' || elevation === '-Infinity')  {
```
Comment (n8rzz) [inline src/assets/scripts/utilities/unitConverters.js]: This is the correct logic, I can see the tests are passing, but I would prefer to combine the logic here with the logic on line 269:
```javascript
if (_isNumber(elevation) || elevation === 'Infinity' || elevation === '-Infinity') {
    return parseFloat(elevation);
}
```

could you also add a comment about why we are looking for strings here?

**Review by n8rzz — CHANGES_REQUESTED** [review]

**Comment by Dabea** [comment]
I added comments and merged the if statements
Also fixed the spelling error and added a link to the original issue.

**Review by n8rzz — DISMISSED** [review]

File: src/assets/scripts/utilities/unitConverters.js [hunk 3]
```diff
@@ -266,7 +266,9 @@ export const parseElevation = (elevation) => {
 
     // if its a number, we're done here.
     // This will catch whole numbers, floats, Infinity and -Infinity.
-    if (_isNumber(elevation)) {
+    // This checks if strings are given will skip the regex and exit early
+    // Also stops the function from returning NaN
+    if (_isNumber(elevation || elevation === 'Infinity' || elevation === '-Infinity')) {
```
Comment (n8rzz) [inline src/assets/scripts/utilities/unitConverters.js]: here it is.  sorry I missed this.  this should actually be:
```javascript
if (_isNumber(elevation) || elevation === 'Infinity' || elevation === '-Infinity') { }
```

**Review by n8rzz — APPROVED** [review]
:shipit:

**Pull request merged**